cryptography/src/appleMain/kotlin/com/wire/kalium/cryptography/MLSClientImpl.kt

```diff
@@ -231,6 +231,10 @@ actual class MLSClientImpl actual constructor(
         TODO("Not yet implemented")
     }
 
+    override fun getGroupVerify(groupId: MLSGroupId): Boolean {
+        TODO("Not yet implemented")
```

Comment:
Should it be implemented now?

I see it's apple, so I guess it'll be solved in the future. But a bit more context would be nice.

```suggestion
        TODO("Not yet supported on apple devices")
```